### ISSUE TITLE
How to install on Android? (Note 8)

### ISSUE THREAD

**Issue by torresjrjr** [tonsky/FiraCode#786]
I have little experience with programming on Android and Linux. Could someone help out?

I have a Samsung Galaxy Note 8, with Android 9 and One UI 1.0.

I wish to use it throughout my phone, and if possible, luckily, on my favourite editor, QuickEdit

**Comment by RubenVerg** [comment]
I don't think you can set custom fonts for system. (Unless hardcoring probably) or maybe there's like a `/fonts` folder from which you can choose system fonts. For quickthing, idkk

**Comment by iwsfutcmd** [comment]
It's practically impossible to add system fonts to Android. You literally have to rebuild the operating system.

If your phone is rooted, there's a hacky way of doing it by putting the appropriate font file in the `/system/fonts` directory and then renaming it to the name of one of the existing fonts. But that's complicated and prone to messing up your system if the right permissions aren't set properly. Sorry :(

**Comment by RubenVerg** [comment]
Also, Galaxy has themes, right? You can become a Samsung Designer or something. Register, download the program and load firacode as main font.

On Aug 13, 2019 23:23, Ben Yang <notifications@github.com> wrote:

It's practically impossible to add system fonts to Android. You literally have to rebuild the operating system.

If your phone is rooted, there's a hacky way of doing it by putting the appropriate font file in the /system/fonts directory and then renaming it to the name of one of the existing fonts. But that's complicated and prone to messing up your system if the right permissions aren't set properly. Sorry :(

**Comment by stephane-archer** [comment]
you can't install fonts on android? it's really strange...
